it/track_repo_compatibility/conftest.py

```diff
@@ -0,0 +1,67 @@
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# 	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+import shlex
+import subprocess
+
+import pytest
+
+RALLY_HOME = os.getenv("RALLY_HOME", os.path.expanduser("~"))
+RALLY_CONFIG_DIR = os.path.join(RALLY_HOME, ".rally")
+RALLY_TRACKS_DIR = os.path.join(RALLY_CONFIG_DIR, "benchmarks", "tracks", "default")
+
+
+@pytest.hookimpl(tryfirst=True)
```

Comment:
Do we need those `hookimp` decorators? Looking at examples of `pytest_addoption`, `pytest_cmdline_main` and `pytest_report_header`, I do not see that used.